### ISSUE TITLE
show block info for deploys

### ISSUE THREAD

**Issue by dckc** [rchain-community/rgov#235]
This is a TODO from 9c537aa55a87

cc @Bill-Kunj

**Comment by Bill-Kunj** [comment]
> This is a TODO from [9c537aa](https://github.com/rchain-community/rgov/commit/9c537aa55a8756e8496fc5f352602b67f746bd8f)
> 
> cc @Bill-Kunj

@dckc Yep, this would be good diagnostic info. Please flesh this out a bit more, though. The link you provide to the commit doesn't really explain anything.

**Comment by Bill-Kunj** [comment]
> for example, a link to the block in the https://revdefine.io explorer...
@dckc I really like https://revdefine.io/. Provides excellent macro-view diagnostics.

> or info about whether the block is finalized
It seems we would have to implement another call to the rnode in participate.js? I'm not sure how to go about that. My knowledge of the API isn't as good as yours.

> or whether executing the code resulted in an error
Kinda thought this is apparent from the results section we display, but maybe it's not. Unless you're talking about the ambiguous ```error deploying rholang code```, which we can't do anything about and, I thought, you were working to resolve with the core team.

> 
> _This is a TODO from [9c537aa](https://github.com/rchain-community/rgov/commit/9c537aa55a8756e8496fc5f352602b67f746bd8f)_
> 
> cc @Bill-Kunj

I think each of these points is large enough to be worthy of an individual issue.
